apple/internal/partials/main_thread_checker_dylibs.bzl

```diff
@@ -56,10 +56,7 @@ def _run_main_thread_checker(
     apple_support.run(
         actions = actions,
         apple_fragment = platform_prerequisites.apple_fragment,
-        arguments = [
-            binary_artifact.path,
```

Comment:
And, confirming, this isn't needed to make it dependent on changes to the `binary_artifact` since it's declared as an `input`? I don't think it is but just want to be certain.

Reply:
Yeah, since it's an `input`, if it changes then this will rerun.